source/SIL.AppBuilder.Portal.Frontend/.babelrc

```diff
@@ -1,10 +1,20 @@
 {
     "plugins": [
+        ["transform-inline-environment-variables", {
+            "include": [
```

Comment:
environment variable white list. only these will get passed on to the bundled js code